### PR TITLE
Mention sharpenedblade/t2linux-fedora-iso for people who need audio or bluetooth.

### PR DESCRIPTION
**Pull request by sharpenedblade**
A lot of people might want bluetooth support, or need audio on first boot for Fedora, so I mentioned a new iso in the Fedora guide.

### CODE REVIEW

File: docs/distributions/fedora/installation.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Download the latest safe release
 
-Many thanks to Mike for building. You can download a live iso [here](https://github.com/mikeeq/mbp-fedora)
+Many thanks to Mike for building. You can download a live iso [here](https://github.com/mikeeq/mbp-fedora).
+
+If you need a more updated kernel, audio working out of the box, LUKS, or bluetooth support use the iso fomr [here](https://github.com/sharpenedblade/t2linux-fedora-iso/releases). Follow the instalation instructions below until step 5, then install like a normal Fedora iso. If you do not know how to install Fedora normally, read [this](https://docs.fedoraproject.org/en-US/fedora/latest/install-guide/). Remember to follow the [wifi guide](https://wiki.t2linux.org/guides/wifi/).
```
Comment (AdityaGarg8) [inline docs/distributions/fedora/installation.md:5]: Spelling of from is written as fomr here. Please correct that.

**Review by AdityaGarg8 — CHANGES_REQUESTED** [review]

File: docs/distributions/fedora/installation.md (hunk 1 above)
Comment (AdityaGarg8) [reply docs/distributions/fedora/installation.md:5]: Also, B capital in Bluetooth, and wifi to be written as Wi-Fi

Comment (AdityaGarg8) [reply docs/distributions/fedora/installation.md:5]: The link now is https://wiki.t2linux.org/guides/wifi-bluetooth/

Comment (sharpenedblade) [reply docs/distributions/fedora/installation.md]: I forgot about this PR, the only benefit of my kernel is faster updates now, so I removed most of that section.

**Review by AdityaGarg8 — APPROVED** [review]

**Pull request merged**